src/common/containers/landing.js

```diff
@@ -59,41 +58,52 @@ class Landing extends Component {
             </div>
           </div>
         </div>
-
-        <section className={ `${styles.section} ${styles.sectionNoBorder} ${containerStyles.lightStrip}` }>
-          <div className={ `${styles.row} ${containerStyles.wrapper}` }>
-            <ListDividedState className={ styles.rowItemGrow }>
-              <li>Scale to millions of installs</li>
-              <li>Available on all clouds and Linux OSes</li>
-              <li>No need for build infrastructure</li>
-            </ListDividedState>
-
-            <ListDividedState className={ styles.rowItemGrow }>
-              <li>Automatic updates for everyone</li>
-              <li>Roll back versions effortlessly</li>
-              <li>FREE for open source projects</li>
-            </ListDividedState>
+        <section className={ `${styles.section} ${styles.sectionTopBorderOnly}` }>
+          <div className={ `${containerStyles.wrapper}` }>
+            <HeadingTwo className={ styles.landingHeading }>
+              Publish your software for
+            </HeadingTwo>
+            <div className={ `${styles.row}` }>
+              <img className={ styles.brandLogo } src={images.debian} />
+              <img className={ styles.brandLogo } src="https://assets.ubuntu.com/v1/59190d38-suse_logo.svg" />
```

Comment:
There's 2 issues here, right? We have some images not on the asset server (I assume they should all be on the asset server) and we have a module of images.  The asset server one I think we all agree on, the module seems like a sane enough thing to me too, so maybe it just needs the image urls updated in the module? Maybe there's a further task to move more images to the module, but that's not for this PR I would say.